packages/core/src/codewhisperer/service/transformByQ/transformMavenHandler.ts

```diff
@@ -108,6 +108,7 @@ function copyProjectDependencies(dependenciesFolder: FolderInfo, modulePath: str
 }
 
 export async function prepareProjectDependencies(dependenciesFolder: FolderInfo, rootPomPath: string) {
+    void vscode.window.showInformationMessage(CodeWhispererConstants.buildStartedNotification)
```

Comment:
We need this here anyway as an unrelated change, but still need to figure out a better solution besides sleeping for half a second

Reply:
It looks like the few chat messages which should be showing up one-at-a-time before the `copyProjectDependencies` and `installProjectDependencies` functions below start, are actually showing up all at once after both of these functions complete. I guess that means these 2 functions are blocking the event loop, which is why adding a small delay before they begin provides enough time for the queued chat messages to send. Haven't been able to find another solution besides adding this delay.

Downsides are that it does delay for all users, and unclear if 500ms will always be enough of a delay.